pkg/generated/restapi/configure_rekor_server.go

```diff
@@ -174,21 +178,81 @@ func setupMiddlewares(handler http.Handler) http.Handler {
 	return handler
 }
 
+type httpRequestFields struct {
+	requestMethod string
+	requestURL    string
+	requestSize   string
+	status        int
+	responseSize  string
+	userAgent     string
+	remoteIP      string
+	latency       string
+	protocol      string
+}
+
+func (h httpRequestFields) MarshalLogObject(enc zapcore.ObjectEncoder) error {
+	enc.AddString("requestMethod", h.requestMethod)
+	enc.AddString("requestUrl", h.requestURL)
+	enc.AddString("requestSize", h.requestSize)
+	enc.AddInt("status", h.status)
+	enc.AddString("responseSize", h.responseSize)
+	enc.AddString("userAgent", h.userAgent)
+	enc.AddString("remoteIP", h.remoteIP)
+	enc.AddString("latency", h.latency)
+	enc.AddString("protocol", h.protocol)
```

Comment:
enc.AddBool("success", h.status >= 200 && h.status < 300)

Reply:
since this can be directly inferred from `status` I think this is redundant